graphql/execute.lua

```diff
@@ -286,6 +286,36 @@ local function getFieldEntry(objectType, object, fields, context)
 
   arguments = setmetatable(arguments, {__index=positions})
 
+  local directiveMap = {}
+  for _, directive in ipairs(firstField.directives or {}) do
+    directiveMap[directive.name.value] = directive
+  end
+
+  local directives = {}
+
+  if next(directiveMap) then
```

Comment:
I suggest to use explicit checks because they protect us from possible box.NULL.

Reply:
I had some painful experience with box.NULL in implicit checks, so I agree with you. But if we've got box.NULL as a key in such a map it would mean that something went completely wrong (and we can't even be sure we need to stop iterating here).

Reply:
Fixed